### PR TITLE
[14.0][MIG] subscription_oca

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
backport from 15.0

### CODE REVIEW

**Comment by francesco-ooops** [comment]
@carlos-domatix @simahawk good for you? :)

**Review by geomer198 — APPROVED** [review]
Code review ok!

File: subscription_oca/__manifest__.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "license": "AGPL-3",
     "author": "Domatix, Odoo Community Association (OCA)",
     "depends": ["sale_management", "account"],
+    "maintainers": ["ilyasProgrammer"],
```
Comment (simahawk) [inline subscription_oca/__manifest__.py:12]: changing maintainer can be a separated commit that you can backport/fwd port easily ;)

**Review by simahawk — APPROVED** [review]
Quickly skimmed through it: LG

**Comment by francesco-ooops** [comment]
@simahawk go go go?

**Comment by damdam-s** [comment]
@ilyasProgrammer @simahawk what the difference between this module and the original `contract` one of this repo? I think it makes a duplicate because this module seems to do (functionally speaking) than `contract` +  `product_contract` together.

I must have missed something important there 😞

**Comment by francesco-ooops** [comment]
@damdam-s the features are similar, but without the complexity of the contract module, eg https://github.com/OCA/contract/pull/971#issuecomment-1652404292

**Pull request merged**